app/consumer-democracy/proposals_whitelisting.go

```diff
@@ -31,19 +29,24 @@ type legacyParamChangeKey struct {
 	Subspace, Key string
 }
 
+// these parameters don't exist in the consumer app -- keeping them as an
```

Comment:
"as an" ? Not sure if something was after "as an" that was accidentaly removed here.